apis/beacon/blocks/blinded_blocks.yaml

```diff
@@ -24,6 +24,9 @@ post:
             - $ref: '../../../beacon-node-oapi.yaml#/components/schemas/SignedBeaconBlock'
             - $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Altair.SignedBeaconBlock"
             - $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Bellatrix.SignedBlindedBeaconBlock"
+      application/octet-stream:
+        schema:
+          description: "SSZ serialized block bytes. Use Accept header to choose this response type, version string is returned in header `Eth-Consensus-Version`."
```

Comment:
you should also declare optional request header under parameters

Reply:
optional request header? oh this is the post... im not sure it's needed, I think I had it and didn't update the header because I couldn't get the syntax to show in swagger but I can either try again or remove that portion of the description on the posts...

Reply:
hopefully my changes have resolved this